.env.template

```diff
@@ -1,2 +0,0 @@
-NEXT_PUBLIC_SUPABASE_URL=YOUR_SUPABASE_URL
-NEXT_PUBLIC_SUPABASE_ANON_KEY=YOUR_SUPABASE_ANON_KEY
```

Comment:
don't delete this file

Reply:
sorry, I renamed the original without making a copy first. would make a PR with changes